utils/data/fuse/good_fuse_targs.yaml

```diff
@@ -133,31 +130,43 @@ DR6:
 - SK-66D100
 - SK-67D216
 - SK-70D13
-- AV435
+- AV-435
 - SK-70D69
 - NGC346-MPG-342
-- AV267
+- AV-267
 - BAT99-105
-- MOA-J010321.3-720538
-- AV26
-- AV235
+- MOA-J010321M720538
+- AV-26
+- AV-235
 - LMCX-4
 - SK-68D16
-- NGC2004-ELS-26
+- NGC2004-ELS-026
 - SK-67D104
 - NGC346-MPG-355
 - SK-65D55
-- AV476
+- AV-476
 - SK-67D106
 - SK-67D168
-- AV39A
+- AV-39A
 - 2DFS-999
-- AV388
-- LH114-7
-- AV83
-- BI272
-# The FUSE data for the targets below was ready for DR6, but HST data was not acquired.
-#DR7:
-#- NGC346-MPG-435
-#- N11-ELS-032
-#- BI13
+- AV-388
+- LH-114-7
+- AV-83
+- BI-272
+DR7:
+- NGC346-MPG-435
+- N11-ELS-032
+- BI-13
+- AV-506
+- BI-128
```

Comment:
This target was dropped from the sample. It should be moved to the bad fuse list